src/Upgrader/Business/Git/CommandResolver/UpdateIndexCommand.php

```diff
@@ -28,6 +28,6 @@ public function isIndexOutdated(): bool
     {
         $commandResultDto = $this->exec($this->getCommand() . ' ' . self::REFRESH_FLAG);
 
-        return $commandResultDto->getResultCode();
+        return boolval($commandResultDto->getResultCode());
```

Comment:
why is this needed? why not a (bool) cast instead?

Reply:
Fixed